src/cjs.js

```diff
@@ -1 +1,4 @@
-module.exports = require('./index').default;
+const fileLoader = require('./index');
+
+module.exports = fileLoader.default;
+module.exports.raw = fileLoader.raw;
```

Comment:
This is mutating `fileLoader.default`... 😕

Maybe it could be better to write the workaround in `index.js` itself, with a comment that it's a workaround for the commonjs export.

They'll both do the same thing, and I guess technically put the cjs change outside of the cjs wrapper, but I'd prefer that to being mutated from outside.